src/test/ui/associated-const/associated-const-in-trait.stderr

```diff
@@ -1,15 +1,17 @@
 error[E0038]: the trait `Trait` cannot be made into an object
   --> $DIR/associated-const-in-trait.rs:9:6
    |
-LL | trait Trait {
-   |       ----- this trait cannot be made into an object...
-LL |     const N: usize;
-   |           - ...because it contains this associated `const`
-...
 LL | impl dyn Trait {
    |      ^^^^^^^^^ the trait `Trait` cannot be made into an object
```

Comment:
this label is now redundant, as the error message and the span is no longer far apart

Reply:
Changed the label to be less verbose, but it is still close to the main message.